frontend/awx/resources/projects/components/CredentialChip.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslation } from 'react-i18next';
+import { Chip } from '@patternfly/react-core';
+import { toTitleCase } from '../../../common/util/strings';
+import { SummaryFieldCredential } from '../../../interfaces/summary-fields/summary-fields';
+
+interface CredentialName {
+  kind: string;
+  name: string;
+  cloud: boolean;
+  inputs?: {
+    vault_id: string;
+  };
+}
+
+export function CredentialChip({
+  credentials: credential,
```

Comment:
Renaming a list of credentials to `credential` can get confusing especially when you're mapping through that list further down.